.github/ISSUE_TEMPLATE/protocol-rfc.md

```diff
@@ -0,0 +1,29 @@
+
+---
+name: Protocol Change Request
+about: Use this template for reporting a bug
+labels: 'protocol'
+title: '[PROTOCOL RFC]'
+
+---
+
+## Protocol Change Request
+
+### Description of the protocol change
+
+<!--
+Please describe the motivation and high-level description of the protocol change you are proposing.
+For a fairly large protocol change, it is recommended that you provide a design doc - (e.g., a google doc, preferably with the ability to comment in the doc). 
+For the next steps on how to proceed with the request, see the protocol RFC process in https://github.com/delta-io/delta/tree/master/protocol_rfcs
```

Comment:
```suggestion
For the next steps on how to proceed with the request, see the protocol RFC process in [https://github.com/delta-io/delta/tree/master/protocol_rfcs](https://github.com/delta-io/delta/tree/master/protocol_rfcs)
```
^^ should turn it into a convenient hyperlink

Reply:
Actually... it will probably render as a hyperlink automatically, just not in the PR diff 🤦

Reply:
Actually neither. Its inside a comment, so it wont render in the markdown form at all. :)